java/arcs/android/storage/service/DevToolsStorageManager.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.storage.service
+
+import arcs.core.storage.StorageKey
+import java.util.concurrent.ConcurrentHashMap
+import kotlin.coroutines.CoroutineContext
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+
+/**
+ * A [StorageServiceManager] is used by a client of the [StorageService] to manage
+ * data stored within the [StorageService].
+ */
+@ExperimentalCoroutinesApi
+class DevToolsStorageManager(
+    /** [CoroutineContext] on which to build one specific to this [StorageServiceManager]. */
+    parentCoroutineContext: CoroutineContext,
+    /** The stores managed by StorageService. */
+    val stores: ConcurrentHashMap<StorageKey, DeferredStore<*, *, *>>
+) : IDevToolsStorageManager.Stub() {
+
+    override fun getStorageKeys(): String {
```

Comment:
At some point I would like to see an separated out interface that describes what messages are allowed to be sent both ways and what are their semantics. For the time being having such custom methods is ok, but I would like us to be more rigorous this time.